app/lib/messages.js

```diff
@@ -1,17 +1,17 @@
-function invalidLocationMessage(location) {
-  return `We can't find the location ${location.toLocaleUpperCase()}. Check the location is correct and try again.`;
+function invalidSearchMessage(search) {
+  return `We can't find the search ${search.toLocaleUpperCase()}. Check the search is correct and try again.`;
 }
 
-function emptyLocationMessage() {
-  return 'You must insert a location to find a surgery.';
+function emptySearchMessage() {
+  return 'You must insert a search to find a surgery.';
 }
 
 function technicalProblems() {
```

Comment:
'the current empty search returns a 500 error, that is something I will be looking next' - you spotted the error, so will fix that now and add a test (where it will be used)